PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -420,7 +474,33 @@ if [ "$CustomBrain" = "NONE" ] ; then
           TXwFolder=${T1wFolder}
           TXwImage=${T1wImage}
           TXwTemplate=${T1wTemplate}
-          TXwTemplate2mm=${T1wTemplate2mm}
+	# Create reference volumes if the resolution of raw image differs from TXwTemplate - TH Mar 2023 
+	StrucRes=$(${FSLDIR}/bin/fslval $(echo ${T1wInputImages} | cut -d ' ' -f1) pixdim1 | awk '{printf "%0.2f", $1}')
+	RefRes=$(${FSLDIR}/bin/fslval ${T1wTemplate} pixdim1 | awk '{printf "%0.2f", $1}')
+	log_Msg "Resolution of structure: $StrucRes"
+	log_Msg "Resolution of T1wTemplate: $RefRes" 
+	if [ ! "$StrucRes" == "$RefRes" ] ; then
```

Comment:
This would trigger on, say, a 0.71mm human scan (or even a 0.7mm if someone specified the 0.8mm template), and use a voxel grid different than the specified template file for all the `MNINonLinear/` volume files.  I'm not sure what we want to happen.